### ISSUE TITLE
Move Mobile Tabs to Bottom

### ISSUE THREAD

**Issue by KevinWu098** [icssc/AntAlmanac#976]
> I do think that the bar should be at the bottom

<img width="486" alt="Screenshot 2024-05-04 at 12 23 30 PM" src="https://github.com/icssc/AntAlmanac/assets/100006999/4daaf49c-24ad-4082-9da3-36c0c220936f">

_Originally posted by @MinhxNguyen7 in https://github.com/icssc/AntAlmanac/pull/921#pullrequestreview-2026748537_

**Comment by vinnyho** [comment]
I will be trying to take this issue.

**Comment by KevinWu098** [comment]
> I will be trying to take this issue.

I believe this one is being worked on in #1001 !

**Comment by MinhxNguyen7** [comment]
> > I will be trying to take this issue.
> 
> I believe this one is being worked on in #1001 !

I wouldn't hold my breath, since @ap0nia has graduated. @vinnyho feel free to take the relevant parts of his code to complete this PR.

**Comment by KevinWu098** [comment]
> > > I will be trying to take this issue.
> > 
> > 
> > I believe this one is being worked on in #1001 !
> 
> I wouldn't hold my breath, since @ap0nia has graduated. @vinnyho feel free to take the relevant parts of his code to complete this PR.

IIRC it's pretty damn close, as far as I can recall

@ap0nia can you confirm? would prefer not to send @vinnyho on the same path if we're close to completion!

**Comment by ap0nia** [comment]
I believe it should almost be done actually. I addressed the feedback that Kevin left on his review, and I was blocked on nitpicks like the name of components.

I can finish writing up the PR summary and QA testing with Kevin if that's alright.

**Comment by vinnyho** [comment]
Thank you, I will find another issue to try and fix. Good luck!
